pages/linux/systemd-run.md

```diff
@@ -0,0 +1,28 @@
+# systemd-run
+
+> Run programs in transient scope units, service units, or path-, socket-, or timer-triggered service units.
+> More information: <https://www.freedesktop.org/software/systemd/man/systemd-run.html>.
+
+- Run a program as root:
+
+`systemd-run {{executable}}`
+
+- Run a program in the user service, keeping the working (d)irectory and waiting until it exits:
+
+`systemd-run --user -d {{executable}}`
+
+- Provide a custom (u)nit name and pass extra parameters:
+
+`systemd-run -u {{unit_name}} -- {{executable}} {{extra parameters ...}}`
```

Comment:
```suggestion
`systemd-run -u {{unit_name}} -- {{executable}} {{extra_parameters ...}}`
```
I am not 100% sure about this, would this be better?

Reply:
I used the current version because `CONTRIBUTING.md` has `{{path/to/directory1 path/to/directory2 ...}}` as an example, and there can be multiple parameters. What do you think about this?

Reply:
> I am not 100% sure about this, would this be better?

According to yours style guide it's incorrect, ellipsis always contain two items, and not one before `...`.

Reply:
I prefer to `{{argument1 argument2 ...}}`.

Reply:
> I prefer to `{{argument1 argument2 ...}}`.

Yeah, seems better